lib/chunkify.js

```diff
@@ -1,13 +1,27 @@
 'use strict';
 
-function chunkify(files, maxChars) {
+function chunkify(files, maxChars, maxBuffer) {
+  const fs = require('fs');
   const filesChunk = [];
   let chunk = '';
+  let chunkLen = 0;
+  let stats;
 
   for (let f = 0, len = files.length; f < len; f++) {
-    if (chunk.length + (files[f].length + 1) > maxChars) {
+    stats = undefined;
```

Comment:
Why do you need to explicitly set it to undefined in the loop?

Reply:
File stats to be available only when file exist (below line), thus making it undefined here to resign the details if file exist.